lib/validation.rb

```diff
@@ -173,6 +173,17 @@ def self.validate_port_range(port_range)
     end_port ? [start_port, end_port] : [start_port]
   end
 
+  def self.validate_port(port)
+    fail ValidationFailed.new({port: "Port must be between 0 to 65535"}) unless port.to_i.to_s == port.to_s
+    fail ValidationFailed.new({port: "Port must be between 0 to 65535"}) unless (0..65535).cover?(port.to_i)
+    port.to_i
+  end
+
+  def self.validate_positive_int(i)
+    fail ValidationFailed.new({i: "Value must be a positive integer."}) unless i.to_i.to_s == i && i.to_i > 0
```

Comment:
The key represents the name of input. Thus, when you define it as `i`, if I remember correctly, the error message won't appear below of the corresponding input.